src/fsharp/FSharp.Core.nuget/FSharp.Core.nuget.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\FSharp.Core\FSharp.Core.fsproj">
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\FSharp.Core\FSharp.Core.fsproj">
```

Comment:
I'm not sure why, but in some places we use `$(MSBuildThisFileDirectory)` and in some we don't.  If it's not needed anywhere we should just remove it everywhere

Reply:
@dsyme,

No, I don't think so.  $(MSBuildThisFileDirectory) always expands to the full path, making ms build logs easier to reason about.  We should prefer the use of the predefined msbuild variables instead of bare relative paths.

Reply:
I don't mind, but TBH it's a bit odd to use the variable here if it's not needed